pkg/templates/manifest.go

```diff
@@ -113,7 +115,7 @@ data:
         value: /tmp/results
       - name: CHROOT_DIR
         value: /node
-      image: gcr.io/heptio-images/sonobuoy-plugin-systemd-logs:latest
+      image: {{.PrivateRegistry}}/sonobuoy-plugin-systemd-logs:latest
```

Comment:
I can see already that this will fail for the normal case - I need to add a specific template variable (or if prefered add a `--systemd-logs-image param`)